src/torrent.cpp

```diff
@@ -10613,15 +10614,17 @@ namespace libtorrent
 			, std::bind(&torrent::on_piece_verified, shared_from_this(), _1)
 			, reinterpret_cast<void*>(1));
 	}
-
-	announce_entry* torrent::find_tracker(tracker_request const& r)
```

Comment:
the whole class ``torrent`` is an implementation detail, so no need to deprecate